test/test_endpoints.py

```diff
@@ -248,3 +255,21 @@ def test_error_response_class_by_hash():
     assert resp.status_code == 500
     expected_message = f"Class with hash {INVALID_HASH} is not declared"
     assert expected_message == error_message
+
+@devnet_in_background()
+def test_get_transaction_status():
+    """Assert valid response schema"""
+    #Create Transaction
+    response = requests.post(f"{APP_URL}/mint", json={
+        "address": "0x0513493b4Fe460031d445fFACacACf3B19196a05Fd146Ed1609B7248101eF847",
+        "amount": 1000e18
+    })
+    assert response.status_code == 200
+    tx_hash = response.json().get("tx_hash")
+
+    json_response = get_transaction_status(tx_hash)
+    assert_valid_schema(json_response, "get_transaction_status.json")
+
+    invalid_tx_hash = "0x443a8b3ec1f9e0c64"
+    json_response = get_transaction_status(invalid_tx_hash)
+    assert_valid_schema(json_response, "get_transaction_status.json")
```

Comment:
Are we happy with just testing the response of a valid and invalid request the same way?

Reply:
What can be a more elegant way?

Testnet return for non-existing address 
```
{
    "tx_status": "NOT_RECEIVED"
}
```
This test checks that we return it also.
Single JSON Schema should provide all cases with a common schema interface for one endpoint.

Reply:
Asserting valid schema is good; perhaps an extra line could be added after the schema assertion that checks the exact value of `"tx_status"`.

Reply:
4c7527bfd06be4f2abdd193ff9de7219b7b69b08